DoSomething/quasar
# Etl monitoring

#### What's this PR do?
Deploys ETL monitoring which is designed to check our key tables for updates on a 24 hour basis

#### Where should the reviewer start?
The bulk of the action is in the ETLMonitoring class

#### How should this be manually tested?
Run the script and see if the slackbot creates a message in the quasar-notifications channel

#### Any background context you want to provide?
We want to know if our key tables are/aren't updating as expected. If there haven't been any new users or signups in a day, something has probably gone wrong and needs to be investigated

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/149589013

#### Next Steps:
Right now, this only tracks the users and campaign activity tables, but the class is designed to very easily add more things to keep an eye on. I'll soon be adding many more, but wanted to get this MVP stood up
